### DISCUSSION TITLE
Add a letter icon ✉️

### DISCUSSION THREAD

**Discussion by mikker** [tailwindlabs/heroicons#637] [Ideas]
I can think of a few use cases… 😅

**Comment by stepanjakl** [comment]
+1 🙏

**Comment by seb-jean** [comment]
There is already: `mail`

**Comment by mikker** [comment]
Perfect. Thanks!

**Comment by stepanjakl** [comment]
Out that's right, I just couldn't see it as I was searching for `email` instead of `mail`. It may be worth implementing a more approximate search technique e.g. https://fusejs.io
